src/CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ set (mraa_LIB_ARM_SRCS_NOAUTO
   ${PROJECT_SOURCE_DIR}/src/arm/adlink_ipi.c
   ${PROJECT_SOURCE_DIR}/src/arm/radxa_zero.c
   ${PROJECT_SOURCE_DIR}/src/arm/radxa_rock3a.c
+  ${PROJECT_SOURCE_DIR}/src/arm/radxa_cm3.c
```

Comment:
Need to rename the file to `radxa_cm3_io.c` and `radxa_cm3_io.h` as well.
You can use `git mv radxa_cm3.c radxa_cm3_io.c` so the git change log will looks nicer.